.travis.d/templates/deployment.yaml.erb

```diff
@@ -34,10 +46,16 @@ spec:
                 value: !!str <%= app['target_port'] or 3000 %>
               - name: ROOT_URL
                 value: <%= "https://#{app['host']}" %>
-              <% if app['wants'] && app['wants']['mongo'] %>
-              - name: MONGO_URL
+              <% if app['wants']%>
+              <% if app['wants']['mongo'] %>
+              - name: <%= app['wants']['mongo'].is_a?(String) ? app['wants']['mongo'] : 'MONGO_URL' %>
                 value: !!str mongodb://<%= biodome['mongo'] %>/<%= app['uid'] %>
               <% end %>
+              <% if app['wants']['postgres'] %>
+              - name: <%= app['wants']['postgres'].is_a?(String) ? app['wants']['postgres'] : 'POSTGRES_URL' %>
```

Comment:
ditto here with `wants.postgres.env`

Reply:
Sidenote: Is that better than `app["wants"]["postgres"]`?